include/fiction/algorithms/path_finding/distance.hpp

```diff
@@ -62,6 +64,30 @@ template <typename Lyt, typename Dist = double>
     return static_cast<Dist>(std::hypot(x, y));
 }
 
+/**
+ * The Euclidean distance between two SiDBs on the H-Si surface (SiQAD coordinates are required) given by
```

Comment:
The documentation does not yet indicate how this function differs from `euclidean_distance`